src/opera/pge/disp_s1/disp_s1_pge.py

```diff
@@ -389,7 +390,48 @@ def _compressed_cslc_filename(self, inter_filename):
             The file name to assign to compressed CSLC product(s) created by this PGE.
 
         """
-        return os.path.basename(inter_filename)
+        level = "L2"
+        name = "COMPRESSED-CSLC-S1"
+
+        ccslc_regex = ("compressed_(?P<burst_id>\w{4}_\w{6}_\w{3})_"
+                       "(?P<ref_date>\d{8})_"
+                       "(?P<start_date>\d{8})_"
+                       "(?P<stop_date>\d{8})[.](?P<ext>h5)$")
```

Comment:
Elegant regex with the capture groups defined within for dictionary access.  Nice!